recipes/sentry-native/all/conandata.yml

```
@@ -8,3 +8,8 @@ sources:
   "0.4.18":
     url: "https://github.com/getsentry/sentry-native/releases/download/0.4.18/sentry-native.zip"
     sha256: "41fdf6499cd8576142beb03104badcc9e0b80b8ef27080ca71cd4408cc1d7ece"
+patches:
+  "0.6.0":
+    - patch_file: "patches/crashpad-wer-support.patch"
+      patch_description: "Make it possible to build with support for the crashpad WER module while using the sentry-crashpad package"
+      patch_type: "conan"
```

Now that the patch has been successfully merged (well done!), it can be added here:
```suggestion
      patch_type: "conan"
      patch_source: "https://github.com/getsentry/sentry-native/pull/816"
```